scripts/build.fish

```diff
@@ -5,8 +5,27 @@ mkdir -p ~/.config/nix
 mkdir -p ~/.config/home-manager
 cp ../nix/nix.conf ~/.config/nix/
 cp ../nix/home.nix ~/.config/home-manager/
+cp ../nix/replace.nix ~/.config/home-manager/
 
 set -x DOTFILES_PATH (realpath ..)
+
+set DOTFILES_USER_CONFIG_PATH ~/.config/dotfiles
+set -x DOTFILES_USER_CONFIG $DOTFILES_USER_CONFIG_PATH/config.json
+mkdir -p $DOTFILES_USER_CONFIG_PATH
+if not test -e $DOTFILES_USER_CONFIG
+  echo \
+'{
+  "HOME": "'$HOME'",
+  "git.user": "example",
+  "git.name": "example",
+  "git.email": "example@gmail.com"
+}'\
```

Comment:
It is quite weird, why a static config template is inlined here?

If I changed the user config for some reason, I may need to modify this inlined template too to reflect the change. This is not good IMO.